### DISCUSSION TITLE
How to adjust the interface in PK Sim & Mobi

### DISCUSSION THREAD

**Discussion by Slow-Walker66** [Open-Systems-Pharmacology/Forum#1353] [Questions & Problems] [answered]
Hello, I find when I use PK Sim and Mobi 11, there are many places not shown clearly in the screen (like the picture displayed below). 

How could I fix this problem?

Thanks in advance!
<img width="401" alt="image" src="https://user-images.githubusercontent.com/87760070/197100461-cd732094-e40d-4730-9114-d0ac693a1492.png">

**Comment by msevestre** [comment, marked as answer]
Hi @1998225 
This is an issue with 4k screens. We are about to release an update in the next few weeks that addresses the issue. 
In the meantime you can either download the portable version (latest built nightly) or maybe try to change your resolution?

**Reply by Slow-Walker66** [reply]
Thanks a lot!
Looking forward to your update.
